runtime/permissions.rs

```diff
@@ -1054,9 +1054,9 @@ fn permission_prompt(message: &str) -> bool {
   if !atty::is(atty::Stream::Stdin) || !atty::is(atty::Stream::Stderr) {
     return false;
   };
-  let opts = "[g/d (g = grant, d = deny)] ";
+  let opts = "[y/n (y = yes allow, n = no deny)] ";
```

Comment:
Any reason to keep deny in the string?

```suggestion
  let opts = "[y/n (y = yes allow, n = no)] ";
```

Reply:
Hmm given that the prompt is a yes/no question, seems redundant and kinda stutters for me but maybe that's just me.

Either way is fine.